### PR TITLE
Initial card block

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

- [x] Add snippet posttype
- [x] Show snippet on a card
- [x] Expand and collapse snippet content

- [x] Merge and publish https://github.com/CodeForAfrica/HURUmap-UI/pull/76

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Desktop Screenshots

Screen Shot 2019-11-05 at 3 28 25 PM

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by DavidLemayian — APPROVED** [review]
Looking good.. :)

File: wp-content/plugins/hurumap/src/config.js [hunk 1]
```diff
@@ -80,7 +80,14 @@ const config = {
       slug: 'zambia',
       published: true
     }
-  ]
+  ],
+  WP_BACKEND_URL:
+    // eslint-disable-next-line no-nested-ternary
+    process.env.NODE_ENV === 'development'
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/config.js:86]: I think we should standardize on checking if `production`...

File: wp-content/plugins/hurumap/src/embed/IFrameContent.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+function IframeContent({ children, expanded }) {
+  const queryParams = new URLSearchParams(window.location.search);
+  const onResize = (_width, height) => {
+    if (window.frameElement) {
+      window.frameElement.style.height = `${height}px`;
+      window.frameElement.style.width = expanded
+        ? '100%'
+        : queryParams.get('width');
+    } else {
+      window.parent.postMessage(
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/embed/IFrameContent.js:16]: :+1:

**Review by kilemensi — APPROVED** [review]
:rocket: @karimkawambwa

**Review by KhadijaMahanga — APPROVED** [review]
:rocket: :rocket:

**Pull request merged**